src/NuGet.Core/NuGet.Packaging/PackageExtractor.cs

```diff
@@ -989,5 +1003,17 @@ public static class PackageExtractor
                 }
             }
         }
+
+        private static RepositorySignatureInfo GetRepositorySignatureInfo(string source)
+        {
+            RepositorySignatureInfo repositorySignatureInfo = null;
+
+            if (!string.IsNullOrEmpty(source))
+            {
+                _repositorySignatureInfoProvider.TryGetRepositorySignatureInfo(source, out repositorySignatureInfo);
```

Comment:
Change this to:

    RepositorySignatureInfoProvider.Instance.TryGetRepositorySignatureInfo(source, out repositorySignatureInfo);